Apps/Inc/UpdateDisplay.h

```diff
@@ -0,0 +1,112 @@
+/** 
+ * @copyright Copyright (c) 2022 UT Longhorn Racing Solar
+ * @file UpdateDisplay.h
+ * @brief Function prototypes for the display application.
+ * 
+ * This contains function prototypes relevant to the UpdateDisplay
+ * application. Call assertUpdateDisplayError after calling any of the
+ * functions in this application.
+ * 
+ * @author Ishan Deshpande (IshDeshpa)
+ * @author Roie Gal (Cam0Cow)
+ * @author Nathaniel Delgado (NathanielDelgado)
+*/
+
+#ifndef __UPDATE_DISPLAY_H
+#define __UPDATE_DISPLAY_H
+
+#include "os.h"
+#include "common.h"
+#include "Tasks.h"
+
+#include "Display.h"
+#include "Contactors.h"
+
+/**
+ * Error types
+ */
+typedef enum{
+	UPDATEDISPLAY_ERR_NONE			= 0,
+	UPDATEDISPLAY_ERR_FIFO_PUT		=-1,	// Error putting command in fifo
+	UPDATEDISPLAY_ERR_FIFO_POP		=-2,	// Error popping command from fifo
+	UPDATEDISPLAY_ERR_PARSE_COMP	=-3,	// Error parsing component/val in SetComponent
+} UpdateDisplayError_e;
```

Comment:
I think this was from an experiment with clang-tidy? I'd like to see it changed back too.

Reply:
This wasn't a clang-tidy thing, I asked Arnav to make sure the code was compliant with the current style guide. I talked with Manthan a little about this on workday but I think we may end up changing it to be _t for enums as well in the style guide. For right now though this is how the style guide wanted it so that's why I did that